libs/blocks/icon-block/icon-block.css

```diff
@@ -42,6 +42,16 @@
   width: 100%;
 }
 
+.five-up .foreground .text-content {
+  text-align: left;
+  padding-bottom: var(--spacing-xs);
+}
+
+
+html[dir='rtl'] .icon-block .foreground .text-content {
+  align-content: right;
```

Comment:
`right` is not a valid property for `align-content` - https://developer.mozilla.org/en-US/docs/Web/CSS/align-content. Looking at previous declarations, maybe you wanted to use `text-align`?

Reply:
This has been fixed, however the whole block was needed.